### ISSUE TITLE
how to train without attribute label

### ISSUE THREAD

**Issue by twmht** [iamhankai/attribute-aware-attention#4]
Hi,

from the table(1) in the paper,  you mentioned that the network can be trained without attribute label.

in such manner,  what is the form of the final loss function? Do we only need to care about the categorization loss?

**Comment by iamhankai** [comment]
just remove the loss of attribute classification, or set lambda to 0 in https://github.com/iamhankai/attribute-aware-attention/blob/ab3210705524bcb56f0121de0bcd47f3f93fb6c5/cub_demo.py#L98

**Comment by twmht** [comment]
very interesting. I am going to do the experiment. How about the other lambda? Do they remain unchanged?

**Comment by iamhankai** [comment]
> very interesting. I am going to do the experiment. How about the other lambda? Do they remain unchanged?

Yes, unchanged.

**Comment by twmht** [comment]
close.
